### ISSUE TITLE
If I have a backend that's integrated with the SSO of my org, should I ask the team to provide a new client Id for the Angular App?

### ISSUE THREAD

**Issue by AbdallahR99** [damienbod/angular-auth-oidc-client#1997]
I'm not familiar with OpenIdConnect/OAuth and I had to integrate my project with my organization's SSO
In the backend (ASP.NET) I did it and handled the callback and everything and made it as an OAuth client and it is that what gets the access token and everything and it has its client id and client secret

now the frontend (Angular) is another service with another domain how to make the angular app to read the authentication if the user is authenticated or his authentication is expired or not and what's his role and claims and so on and the right way to do it

so I don't want to generate now client id and client secret for the Angular App service but I want it the same as the backend one and to be integrated together and once it done from the backend should be passed to the frontend

I need the backend to handle the SSO authentication integration like getting the token and user info and registering it in the database if a first time or some actions and just passing it to the frontend 

but I need the right way to do, like if the SSO token is expired or something the Angular App must be aligned up with that 

in short how to pass the authentication from the backend (which is just a client) to the frontend
